src/Webapi/Dom/Webapi__Dom__Window.re

```diff
@@ -4,7 +4,6 @@ type frameList; /* array-like, WindowProxy? */
 type idleDeadline; /* Cooperative Scheduling of Background Tasks */
 type locationbar; /* "bar object" */
 type menubar; /* "bar object" */
-type navigator;
```

Comment:
Deleting this type will break backward-compatibility, so instead can you alias it to the newly-introduced `Webapi__Dom__Navigator.t` type and deprecate the alias, thx